editor.planx.uk/src/pages/Preview/Node.tsx

```diff
@@ -172,6 +173,9 @@ const Node: React.FC<any> = (props: Props) => {
     case TYPES.List:
       return <List {...allProps} />;
 
+    case TYPES.MapAndLabel:
+      return <MapAndLabelComponent />;
```

Comment:
Looks like this is missing `{...props}` ! Passing these in should allow us to access/display title in graph etc